RadixWalletTests/PreludeTests/IdentifiedArrayOf+Mutation+Position.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+@testable import Radix_Wallet_Dev
+import Sargon
+import XCTest
+
+final class IdentifiedArrayOfTests: XCTestCase {
+	// https://github.com/pointfreeco/swift-identified-collections/pull/66
+	func testIdentifiedArraySubscript() {
```

Comment:
this new test failed before bumping to `1.0.1`. Prooving that `1.0.1` indeed fixes it for us. 

It is a copy paste (then translate) of https://github.com/pointfreeco/swift-identified-collections/pull/66/files#diff-e480af6c6a4ba50def3412409512e05342d90727cf883bb6d2186f123f204b13R376